docs/LogGuide.md

```diff
@@ -845,39 +1306,145 @@ Marker Code | Name | Sample Locations | Consistent meaning?
+29|2021-06-10T20:15:15.1000000-04:00|Add|0|10FF0001|Tini Poutini|400075A9|||0afeb301004fc6d8c7bd5f6cbda9dce7
+29|2021-05-25T22:54:32.5660000-04:00|Add|6|10FF0001|Tini Poutini|10FF0002|Potato Chippy||70a8c8a728d09af83e0a486e8271cc57
+
+ACT Log Line Examples:
+[20:15:15.100] 1D:29:2021-06-10T20:15:15.1000000-04:00:Delete:0:10FF0001:Tini Poutini:4000641D:::50460af5ff3f8ec9ad03e6953d3d1ba9
+[20:15:15.100] 1D:29:2021-06-10T20:15:15.1000000-04:00:Add:0:10FF0001:Tini Poutini:400075A9:::0afeb301004fc6d8c7bd5f6cbda9dce7
+[22:54:32.566] 1D:29:2021-05-25T22:54:32.5660000-04:00:Add:6:10FF0001:Tini Poutini:10FF0002:Potato Chippy::70a8c8a728d09af83e0a486e8271cc57
+```
+
+<!-- AUTO-GENERATED-CONTENT:END -->
 
 ### 1E: NetworkBuffRemove
 
 This is the paired "end" message to the [1A: NetworkBuff](#1a-networkbuff) "begin" message.
 This message corresponds to the loss of effects (either positive or negative).
 
-Structure:
-`1E:[ObjectId]:[Target Name] loses the effect of [Status] from [Source Name]`
+<!-- AUTO-GENERATED-CONTENT:START (logLines:type=networkTargetMarker&lang=en) -->
 
-Examples:
+#### Structure
 
 ```log
-1E:10657868:Tini Poutini loses the effect of Sprint from Tini Poutini.
-1E:10299838:Potato Chippy loses the effect of Protect from Tater Tot.
-1E:40686258:Ovni loses the effect of Aero II.
+Network Log Line Structure:
+29|2021-04-26T14:11:35.0000000-04:00|[operation]|[waymark]|[id]|[name]|[targetId]|[targetName]
+
+ACT Log Line Structure:
+[14:11:35.0] 1D:29:2021-04-26T14:11:35.0000000-04:00:[operation]:[waymark]:[id]:[name]:[targetId]:[targetName]
 ```
 
+#### Examples
+
+```log
+Network Log Line Examples:
+29|2021-06-10T20:15:15.1000000-04:00|Delete|0|10FF0001|Tini Poutini|4000641D|||50460af5ff3f8ec9ad03e6953d3d1ba9
+29|2021-06-10T20:15:15.1000000-04:00|Add|0|10FF0001|Tini Poutini|400075A9|||0afeb301004fc6d8c7bd5f6cbda9dce7
+29|2021-05-25T22:54:32.5660000-04:00|Add|6|10FF0001|Tini Poutini|10FF0002|Potato Chippy||70a8c8a728d09af83e0a486e8271cc57
+
+ACT Log Line Examples:
+[20:15:15.100] 1D:29:2021-06-10T20:15:15.1000000-04:00:Delete:0:10FF0001:Tini Poutini:4000641D:::50460af5ff3f8ec9ad03e6953d3d1ba9
+[20:15:15.100] 1D:29:2021-06-10T20:15:15.1000000-04:00:Add:0:10FF0001:Tini Poutini:400075A9:::0afeb301004fc6d8c7bd5f6cbda9dce7
+[22:54:32.566] 1D:29:2021-05-25T22:54:32.5660000-04:00:Add:6:10FF0001:Tini Poutini:10FF0002:Potato Chippy::70a8c8a728d09af83e0a486e8271cc57
+```
+
+<!-- AUTO-GENERATED-CONTENT:END -->
+
 ### 1F: NetworkGauge
 
 Info about the current player's job gauge.
 
-Examples:
+<!-- AUTO-GENERATED-CONTENT:START (logLines:type=networkGauge&lang=en) -->
+
+#### Structure
 
 ```log
-1F:10532971:Tini Poutini:C8000019:FD32:D0DF8C00:7FC0
-1F:10532971:Potato Chippy:C863AC19:1000332:D0DF8C00:7FC0
+Network Log Line Structure:
+31|2021-04-26T14:11:35.0000000-04:00|[id]|[data0]|[data1]|[data2]|[data3]
+
+ACT Log Line Structure:
+[14:11:35.0] 1F:[ID]::[data0]:[data1]:[data2]:[data3]
```

Comment:
Is there an extra colon here, or is this how it looks?

Reply:
Those blank fields should be the player name but that info isn't on the network line. Maybe it makes sense to pre-populate the `LogRepository` instance with the generic combatant info?